docsite/source/custom-predicates.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Custom predicates
+layout: gem-single
+name: dry-logic
+---
+
+Define custom predicates using the `predicate` method.
+
+``` ruby
+build do
+  predicate :divisible_with? do |num, input|
+    (input % num).zero?
+  end
+end
+```
+
+The first value `num` to `predicate` is defined by `divisible_with?(num)` and the second argument `input` at a later point by the caller.
+
+``` ruby
+is_divisible_with_ten = build do
+  divisible_with?(10)
+end
+
+is_divisible_with_ten.call(10) # => true
```

Comment:
`call` doesn't return `true`/`false` but a result object. Is this something that has to be clarified?

Reply:
Oh for sure, this is a mistake in the docs.

Reply:
Or should I just have "call" return a bool? Maybe it would be a good idea to abstract away the result type.

Reply:
Fixed in 0834b85.